ACE/include/makeinclude/platform_android.GNU

```diff
@@ -139,10 +139,12 @@ rwho = 0
 #$TOOLCHAIN/bin in path, $TOOLCHAIN is root of standalone toolchain
 #
 #Optionally ANDROID_ABI can be set to select specific architecture ABIs like armeabi-v7a or neon
+ANDROID_ABI ?= arm64-v8a
```

Comment:
Necessary for ABI specific flags. Was unfamiliar with what can go into platform_macros.GNU. I'll work on this.